src/parsers/parsers.js

```diff
@@ -1,11 +1,41 @@
 import cif from "./cif";
+// eslint-disable-next-line import/no-cycle
 import espresso from "./espresso";
 import poscar from "./poscar";
+// eslint-disable-next-line import/no-cycle
```

Comment:
we have circular dependencies with the xyz parser that we need to address in the future. @tjduigna suggested using this approach in the meantime until we can clean things up.